cors/resources/access-control-expose-headers.json

```diff
@@ -0,0 +1,62 @@
+[
+  {
+    "input": "access-control-expose-headers: BB-8",
+    "exposed": true
+  },
+  {
+    "input": "Access-Control-Expose-Headers: bb-8,,@#$#%%&^&^*()()11!",
+    "exposed": false
+  },
+  {
+    "input": "Access-Control-Expose-Headers: bb-8, no no",
+    "exposed": false
+  },
+  {
+    "input": "Access-Control-Expose-Headers: @#$#%%&^&^*()()11!,bb-8",
+    "exposed": false
+  },
+  {
+    "input": "Access-Control-Expose-Headers: bb-8\r\nAccess-Control-Expose-Headers: no",
+    "exposed": true
+  },
+  {
+    "input": "Access-Control-Expose-Headers: bb-8\r\nAccess-Control-Expose-Headers: no no",
+    "exposed": false
+  },
+  {
+    "input": "Access-Control-Expose-Headers: no\r\nAccess-Control-Expose-Headers: bb-8",
+    "exposed": true
+  },
+  {
+    "input": "Access-Control-Expose-Headers:\r\nAccess-Control-Expose-Headers: bb-8",
+    "exposed": true
+  },
+  {
+    "input": "Access-Control-Expose-Headers: ,bb-8",
+    "exposed": true
+  },
+  {
+    "input": "Access-Control-Expose-Headers: bb-8\u000C",
+    "exposed": false
+  },
+  {
+    "input": "Access-Control-Expose-Headers: bb-8\u000B",
+    "exposed": false
+  },
+  {
+    "input": "Access-Control-Expose-Headers: bb-8\u000B,bb-8",
+    "exposed": false
+  },
+  {
+    "input": "Access-Control-Expose-Headers: 'bb-8'",
+    "exposed": false
+  },
+  {
+    "input": "Access-Control-Expose-Headers: 'bb-8',bb-8",
+    "exposed": true
+  },
+  {
+    "input": "Access-Control-Expose-Headers: \"bb-8\",bb-8",
+    "exposed": false
+  }
+]
```

Comment:
I've been reading the linked HTTP ABNF RFC (https://tools.ietf.org/html/rfc7230#section-3.2.6), which does seem define about the thing browsers do, expect the `"` DQUOTE wrapped content should allow you to send any-random thing through.

So proposing adding these tests, one positive test for all valid text:
```
  {
    "input": "Access-Control-Expose-Headers: valid_!#$&'*+-.^`|~1a%40z,bb-8",
    "exposed": true
  },
```
Two on the browser-specific (checked gecko and chromium (Opera + Chrome)) ignore of what the spec (as I read it which is probably wrong) allows:
```
 {
    "input": "Access-Control-Expose-Headers: \"bb-8\"",
    "exposed": false
  },
  {
    "input": "Access-Control-Expose-Headers: bb-8\r\nAccess-Control-Expose-Headers: \"no no\"",
    "exposed": false
  },
```
If we *would* want to follow the spec (at least as I read it), we could just flip `exposed: true` on those two (the tests existing would still fail as I read it).

Reply:
What makes you think this header allows quoted strings?

Reply:
Hm. I was confused. But I can explain.

Spec links to https://tools.ietf.org/html/rfc7230#section-3.2
I was reading `3.2.6 Field Value Components`: https://tools.ietf.org/html/rfc7230#section-3.2.6
It has the section on `quoted-string`, but Fetch doesn't actually specify that, and the "default" 3.2 also does not specify it.

The tests could still be included as a "don't do this" though, so no-one does the same error as me. But other people might be less likely to do such wrong readings though :)